client/components/Signup.js

```diff
@@ -71,8 +76,8 @@ export class Signup extends Component {
             <button type="submit">Submit</button>
           </div>
             {
-              this.props.error &&
-                <div>ERROR: An account with this email already exists. Please login through the login portal.</div>
+              this.props.error.status &&
```

Comment:
ideally we want to be consistent throughout, other components we deconstructed, not here, jsut a note